CHANGELOG.md

```diff
@@ -13,16 +13,16 @@ version directory, and  then changes are introduced.
 
 ## [Unreleased]
 
+### Fixed
 
+- better error handling for filemanager
```

Comment:
I think we can skip changelog here. We don't know the bug it fixes so it isn't really interesting for the end user.